translations/zh_CN.ts

```diff
@@ -657,8 +657,13 @@ git 提交编号: %3
         <translation>未能成功运行。请先编译。</translation>
     </message>
     <message>
-        <source>Please install xterm in order to use Detached Run.</source>
-        <translation>请安装 xterm 以使用在终端中运行。</translation>
+        <source>%1 could not be found. You can change the terminal from Preferences -&gt; Action -&gt; Detached Execution.</source>
```

Comment:
Please, update Chinese translations

Reply:
He'll update it this time. I think he waited because he knew that some changes were to be brought.